core/package.json

```diff
@@ -1,27 +1,20 @@
 {
-  "name": "webviz-plotly",
-  "version": "0.1.0",
-  "description": "Webviz page element for plotly visualizations.",
+  "name": "webviz",
+  "version": "0.10.1",
```

Comment:
why bump to this version number?

Reply:
This is the version set in setup.py, 0.1.0 is simply outdated.